src/backend/dist.js

```diff
@@ -365,7 +365,7 @@ module.exports = {
 
     try {
       var fileData = fs.readFileSync(filePath).toString().split('\n');
-      fileData.unshift("{\n\"short_name\":\"" + eventName+"\",");
+      fileData.unshift("{\n\"name\":\""+eventName+"\",\n\"short_name\":\"" + eventName+"\",");
```

Comment:
> fileData.unshift("{\n\"name\":\""+eventName+"\",\n\"short_name\":\"" + eventName+"\",");
Why do we need to write it as `name:` @agbilotia1998

Reply:
Please check the issue #1575 and the attached screenshot.

Reply:
@aayusharora We are dynamically adding the name of the event to manifest.Please check screenshot
![screen shot 2018-01-09 at 12 37 24 am](https://user-images.githubusercontent.com/12807846/34687852-fe4b6a24-f4d5-11e7-8dd0-f5cee0e712aa.png)